api-report/purchases-js.api.md

```diff
@@ -277,6 +285,8 @@ export interface PurchaseParams {
     customerEmail?: string;
     defaultLocale?: string;
     htmlTarget?: HTMLElement;
+    // Warning: (ae-forgotten-export) The symbol "PurchaseMetadata" needs to be exported by the entry point Purchases.es.d.ts

```

Comment:
Seems we need to export the type from `main.ts`?

Reply:
Nice catch, fixed it!